common/main/en.xml

```diff
@@ -152,7 +152,6 @@ annotations.
 			<language type="csw">Swampy Cree</language>
 			<language type="cu">Church Slavic</language>
 			<language type="cv">Chuvash</language>
-			<language type="cwd">Woods Cree</language>
```

Comment:
Do we need to remove these? I imagine they are considered invalid because`cr` & `hai` are macrolanguages -- since macrolanguages cannot have locale data (it's poorly defined) `cwd` is aliased to `cr` and `hdn` is aliased to `hai`.

However it depends on how this data is used. If its for translations of ISO names people will want these constituent languages. Additionally `cr` above is `<language type="cr">Cree</language>` -- contributing to the further confusion about what our `cr` data actually is.

imho we should keep `cwd` here. Additionally (I'm happy to file a ticket if you like the idea) we should move the current `cr.xml` to `cwd.xml` and leave `cr.xml` as a stub that points to `cwd.xml` -- that will make the macrolanguage/alias hierarchy much clearer to external users of the CLDR data.

cc @srl295 @DavidLRowe who also had a lot of context on the first nations macrolanguages.

Reply:
Normally we use the common name, without terms to distinguish from other variants. So we say "Arabic" and not "Modern Standard Arabic". We do have the facility for the long form, however; you see that for zh.

So we could have 
<language type="cr">Cree</language>
<language type="cr" alt="long">Woods Cree</language>

That is consistent with other cases, so I can add that.

I disagree with the stub approach. We do that for 'no' and 'en-001' and it is a pain.

Reply:
Yeah, I don't think removing encompassed languages makes sense here.  cwd is a valid display name and should be retained.

Reply:
See below